script/dump-db

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -euo pipefail
-DUMP_FOLDER=${DUMP_FOLDER:-"/var/lib/openqa/SQL-DUMPS"}
+DUMP_FOLDER=${DUMP_FOLDER:-"/var/lib/openqa/backup"}
```

Comment:
This breaks expectation of how o3 and osd handle backups. As I expect at this point it is only o3 and osd using this newly added script I think it's ok if we change it but we should be explicit and prepare for that. So please put this into a separate commit and adapt o3 and osd backup at least in https://gitlab.suse.de/qa-sle/backup-server-salt/-/blob/master/rsnapshot/rsnapshot.conf?ref_type=heads#L28 and https://gitlab.suse.de/openqa/salt-states-openqa/-/blob/master/etc/backup/rsnapshot_generic.conf?ref_type=heads#L28 and accordingly for OSD, etc.

Reply:
I think it won't break it. We are not deleting existing symlink. So, `SQL-DUMPS` will continue to follow `backup` directory. If backup directory gets updated the same will be reflected in the `SQL-DUMPS`

Reply:
I have created merge requests.
- https://gitlab.suse.de/qa-sle/backup-server-salt/-/merge_requests/25/diffs
- https://gitlab.suse.de/openqa/salt-states-openqa/-/merge_requests/1457/diffs